app/models/katello/content_view.rb

```diff
@@ -18,6 +18,7 @@ class ContentView < Katello::Model
 
     has_many :content_view_versions, :class_name => "Katello::ContentViewVersion", :dependent => :destroy
     alias_method :versions, :content_view_versions
+    has_one :latest_version_object, -> { latest }, :class_name => "Katello::ContentViewVersion"
```

Comment:
The `latest` method you have added to `content_view_version.rb` returns all the versions, sorted. But `latest_version_object` returns only one record, which surprised me. I guess Active Record is smart enough to see `has_one` and return only one?

Reply:
I think I found the answer to my own question [in Active Record](https://github.com/rails/rails/blob/68eade83c87ae309191add6dfa4959d7d7e07464/activerecord/lib/active_record/associations/association_scope.rb#L30)